src/components/quiz-question/open-question/open-question.tsx

```diff
@@ -30,7 +31,7 @@ export const OpenQuestion = withText(translates)(
     const quizQuestionRef = useRef<HTMLLegendElement>(null);
     useEffect(() => {
       if (!disabled) {
-        quizQuestionRef.current?.focus();
+        textareaRef.current?.focus();
```

Comment:
it looks like `quizQuestionRef` doesn't uses anymore, let's remove it then

Reply:
done